posthog.go

```diff
@@ -238,17 +250,28 @@ func (c *client) Enqueue(msg Message) (err error) {
 	return
 }
 
+func (c *client) setLastCapturedEvent(event Capture) {
+	c.lastEventMutex.Lock()
+	defer c.lastEventMutex.Unlock()
+	c.lastCapturedEvent = &event
+}
+
+func (c *client) GetLastCapturedEvent() *Capture {
+	c.lastEventMutex.RLock()
+	defer c.lastEventMutex.RUnlock()
+	if c.lastCapturedEvent == nil {
+		return nil
+	}
+	// Return a copy to avoid data races
+	eventCopy := *c.lastCapturedEvent
+	return &eventCopy
+}
```

Comment:
this is a handy method for tracking captured events, but I mostly only used it for testing.  Perhaps I shouldn't expose it on the API client?

Reply:
eh, should be fine. Doesn't seem too unsafe to expose